samwise.js

```diff
@@ -1,13 +1,15 @@
-const randomIndex = Math.floor(Math.random() * array.length);
```

Comment:
SCRIPT-I'm a bit concerned about the potential performance impact of this loop. Have you thought about optimizing it?